RawImgFrameSpecs fb
uint32 category
uint32 instanceNum
int32 sequenceNum
depthai_common_msgs/Timestamp ts
uint8[] data
